nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/service/impl/RequestProcessor.java

```diff
@@ -126,6 +138,29 @@ public void processRequest(IStorageRequest request) {
         }
     }
     
+    @Override
+    public void processRequest(IStorageDeletionRequest request) throws NepomukFileStorageException {
```

Comment:
this method had to handle the exceptions. if it doesn't then it never sends an error response to giles and giles won't know about a failed processing.

Reply:
or at the least it needs to send a message via the message handler.